cmd/remote.go

```diff
@@ -35,6 +35,25 @@ func Remote(c *git.Client, args []string) error {
 		}
 		aopts := git.RemoteAddOptions{opts}
 		return git.RemoteAdd(c, aopts, args[1], args[2])
+	case "get-url":
+		uflags := newFlagSet("remote-get-url")
+		urlopts := git.RemoteGetURLOptions{RemoteOptions: opts}
+		uflags.BoolVar(&urlopts.Push, "push", false, "Print push URLs, not fetch URLs")
+		uflags.BoolVar(&urlopts.All, "all", false, "Print all URLs, not just the first")
```

Comment:
The 'all' option doesn't seem to be used here. Maybe the description should indicate that it is not yet supported?

Reply:
sure, done..